src/@types/alltypes.d.ts

```diff
@@ -19,24 +20,24 @@ type VariateComponentProps = {
 }
 
 type ExperimentType = {
-  attributes: object,
-  bucket: number,
-  experiment: number,
+  attributes: object
+  bucket: number
+  experiment: number
   variation: number
 }
 
-interface ComponentReturnInterface {
-  componentName: string,
-  experiments: Array<ExperimentType>,
+type ComponentReturnType = {
+  componentName: string
+  experiments: Array<ExperimentType>
```

Comment:
Is this `ExperimentType` different than the `Experiment` class defined in the engine?

Reply:
It should be the same as the one in the engine.